Assets/Scripts/Hex/HexVertices.cs

```diff
@@ -0,0 +1,239 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using UnityEngine;
+
+/// <summary>
+/// A list of Hex types supported by this package.
+/// </summary>
+public enum HexTypes
+{
+    FlatTop,
+    PointyTop
+}

```

Comment:
Consider putting this in its own file. There's nothing really tying this enum to this class.

Reply:
Moved